webauthn-rs/src/lib.rs

```diff
@@ -199,6 +199,11 @@ use webauthn_rs_core::WebauthnCore;
 
 use crate::interface::*;
 
+/// Fake `CredentialID` generator. See [WebauthnFakeCredentialGenerator] for more details.
```

Comment:
`See [WebauthnFakeCredentialGenerator](fake::WebauthnFakeCredentialGenerator) for more details.`